src/shared/components/palette/base-field/field-read.component.ts

```diff
@@ -31,6 +31,8 @@ export class FieldReadComponent extends AbstractFieldReadComponent implements On
     let injector = Injector.create([], this.fieldContainer.parentInjector);
     let component = this.resolver.resolveComponentFactory(componentClass).create(injector);
 
+    console.log('this.caseField=', this.caseField);
```

Comment:
Missed to remove these console logs before committing?

Reply:
Done